cli/remote.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"log"
+	"os/exec"
+
+	"github.com/flynn/flynn/Godeps/_workspace/src/github.com/flynn/go-docopt"
+	"github.com/flynn/flynn/controller/client"
+)
+
+func init() {
+	register("remote", runRemote, `
+usage: flynn remote add [<remote>] [-y]
+
+Create a git remote that allows deploying the application via git.
+If a name for the remote is not provided 'flynn' will be used.
+
+Options:
+	-y, --yes              Skip the confirmation prompt if the git remote already exists.
+
+Examples:
+
+	$ flynn remote add turkeys-stupefy-perry
```

Comment:
I think this remote name should be `staging` or similar and reflected in the line below, currently it looks like an app name (maybe move that to demonstrate the `-a` flag).